recipe/build_opencv.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+set -ex
+
+# CMake FindPNG seems to look in libpng not libpng16
+# https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindPNG.cmake#L55
+ln -s $PREFIX/include/libpng16 $PREFIX/include/libpng
+
+V4L="1"
+
+if [[ "${target_platform}" == linux-* ]]; then
+    # Looks like there's a bug in Opencv 3.2.0 for building with FFMPEG
```

Comment:
is it relevant to v4.10.0?

Reply:
Looks like not, but I am pretty sure the flag has no effect.

Reply:
yep